components/script/dom/scrolling_box.rs

```diff
@@ -128,15 +128,11 @@ impl ScrollingBox {
     }
 
     pub(crate) fn can_keyboard_scroll_in_axis(&self, axis: ScrollingBoxAxis) -> bool {
-        let axes_overflow = match &self.target {
-            ScrollingBoxSource::Element(_, axes_overflow) => axes_overflow,
-            ScrollingBoxSource::Viewport(_) => return true,
-        };
         let overflow = match axis {
-            ScrollingBoxAxis::X => axes_overflow.x,
-            ScrollingBoxAxis::Y => axes_overflow.x,
+            ScrollingBoxAxis::X => self.overflow.x,
+            ScrollingBoxAxis::Y => self.overflow.x,
         };
-        if !overflow.is_scrollable() || overflow == Overflow::Hidden {
+        if overflow == Overflow::Hidden {
```

Comment:
this would return true for Overflow::Clip, which i think won’t happen due to how the method is called, but maybe we want to assert that invariant?